docs/DeveloperGuide.adoc

```diff
@@ -218,6 +218,46 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+// tag::removeTag[]
+=== removeTag command
+
+The `removeTag` command is to remove a tag with name specified from address book. All the appearances of the tag in
```

Comment:
"The `removeTag` command removes a tag with the specified name from the address book."